src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -1005,7 +999,8 @@ private PreparedStatement getUpdateStreamFromDb(Connection conn)
     GSA_DENY_USERS,
     GSA_PERMIT_GROUPS,
     GSA_DENY_GROUPS,
-    GSA_TIMESTAMP;
+    GSA_TIMESTAMP,
+    GSA_ACTION
```

Comment:
I habitually add new enums to the end for backward compatibility issues. But there are none here.